.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: python
-sudo: false
+sudo: required
```

Comment:
This is needed - the [docs](https://docs.travis-ci.com/user/trusty-ci-environment/) state "all Trusty builds have sudo enabled".